client/state/ui/action-log/selectors.js

```diff
@@ -30,6 +30,6 @@ export function getActionLog( state ) {
  * @return {Object}              The matching dispatched action
  */
 export const getLastAction = createSelector(
-	( state ) => last( state.ui.actionLog ),
+	( state ) => last( state.ui.actionLog ) || false,
```

Comment:
This is throwing an error now (see CircleCI), so it seems like you'll need to modify a test too 😄.